web/templates/web/study-detail.html

```diff
@@ -36,6 +37,11 @@
         function childSelected(selected) {
             var participateButton = document.getElementById('participate-button');
             selected.value === 'none' ? participateButton.disabled = true : participateButton.disabled = false;
+            participateButton.value = selected.value;
+        }
+
+        function portToParticipate(selected) {
```

Comment:
Might be worth doing this in a `window.open` so they can preview while having their experiment in another tab for editing but just an idea.